homeassistant/components/dynalite/config_flow.py

```diff
@@ -31,28 +20,13 @@ def __init__(self):
     async def async_step_import(self, import_info):
         """Import a new bridge as a config entry."""
         LOGGER.debug("async_step_import - %s", import_info)
-        host = self.context[CONF_HOST] = import_info[CONF_HOST]
+        host = import_info[CONF_HOST]
+        await self.async_set_unique_id(host)
+        self._abort_if_unique_id_configured()
         return await self._entry_from_bridge(host)
```

Comment:
We want to test that the connection to the bridge works for this host, before creating the entry.

Reply:
this will be quite tricky since if I create the bridge here, I cannot pass it to the async_setup_entry. I looked in the hue component that is similar, and they create a bridge to try and connect and then another one when it is set up, which I can also do, although it will include some code duplication. 

Is this something that would be helpful? As this is coming from static configuration.yaml, it may be cleaner to leave as is. Your call

Reply:
ok. I did something. initially it works by polling (every second for 30 seconds then times out). I figured it could only cause a short delay the first time we configure, so it should be ok to any user

Reply:
We want to create a bridge here and then a new bridge in `async_setup_entry`. That's our standard way.

Reply:
ok. this is what i did (hopefully...)